implementations/rust/ockam_macros/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "ockam_macros"
+version = "0.1.0"
+authors = ["Ockam Developers"]
+edition = "2018"
+
+[lib]
+proc-macro = true
+
+[features]
+default = []
+async = []
+executor_async = ["async", "executor"]
+
+[dependencies]
+executor = {version = "*", optional = true}
```

Comment:
could you add a trailing empty line to the file
our `.editorconfig` file specifies that but we don't have a linter for it yet.